### ISSUE TITLE
Code review for ove-core

### ISSUE THREAD

**Issue by jamesscottbrown** [dsi-icl/ove#84]
## src/client/utils/utils.js

1. [utils.js](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/utils/utils.js#L184) refers to a ``canvas`` in several places, when I think that what is meant is ``section``. To avoid confusion, the code shouldn't use ``canvas`` to refer to anything other than an HTML [``<canvas>``](https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API) element.

2. The``getLogLabelCSS()`` function can be significantly simplified by  [switch statement on logLevel](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/utils/utils.js#L48) could be eliminated by modifying the ``LogPrefix`` list to record colours as well as names (or recording the colours in a separate dictionary):

````JavaScript
        const LogLevelDetails = {
            TRACE: {level: 'TRACE', background: '#808080', color: '#FFFAF0'},
            ...
        };

        ...

        const getLogLabelCSS = function (logLevel) {
             return 'font-weight: 500; background: ' + LogLevelDetails[logLevel].background + '; color: ' + LogLevelDetails[logLevel].color;
       }
````

3. Similarly, the [six logging functions](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/utils/utils.js#L81) have essentially the same structure, so could be generated by iterating over the list of log types and attaching functions to the ``this`` object.

4. Why do [``initControlOnDemand``](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/utils/utils.js#L146) and ``initControl`` log errors with different levels?

5. I think the comments in [``resizeController``](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/utils/utils.js#L146) would be clearer as 

       // The element with id contentDivName is scaled to fit inside both the client
       // and window, whilst maintaining the aspect ratio of the section/content.

and that the comments in [``resizeViewer``]() would be clearer as 

       // The element with id contentDivName is resized to the size of the corresponding
       // section (which may span multiple clients), and then translated based on the 
       // client's coordinates.

6. ``Internal Utility function to get logger arguments`` would be more accurately stated as ``Internal Utility function to construct error messages``. The name is also misleading: ``getArgsToLog`` suggests it simply gathers arguments, rather than constructing the actual log message: something like ``getLogMessage`` would be better.

## src/client/ove.js

1. What is ["useful for opening WebSockets"](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/ove.js#L14) - including or ignoring the scheme?

2. In [this comment](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/ove.js#L121), ``{space}-{client}.{sectionId}``, ``client`` should be ``clientId`` for consistency (and ``space`` should probably be ``spaceId``, but this name is used only in the alignment app)

## src/client/view/core.js

1. Don't specify the exact time in the comments  ["after 15s"](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/view/core.js#L15) and ["for a further 5s"](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/view/core.js#L24): this will become wrong if the value of the constant is changed

2. ["scaled to suit display ranges in most screens"](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/view/core.js#L51) is a bit mysterious

3. [`` // All sections can be deleted at once.``](https://github.com/dsi-icl/ove/blob/0bf84290f5e9f51005bea63e17c5ac50ba37036f/packages/ove-core/src/client/view/core.js#L87) is redundant as a comment, as the logging message on the line above says the same thing

**Comment by jamesscottbrown** [comment]
The comments on logging above apply also to ``ove-lib/utils/src/index.js``

**Comment by senakafdo** [comment]
Everything node here is fixed, and the issue on naming is moved to #25.
